x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/async/AsyncTaskIndexService.java

```diff
@@ -50,16 +51,13 @@
 
 import static org.elasticsearch.common.xcontent.XContentFactory.jsonBuilder;
 import static org.elasticsearch.index.mapper.MapperService.SINGLE_MAPPING_NAME;
-import static org.elasticsearch.xpack.core.ClientHelper.ASYNC_SEARCH_ORIGIN;
 import static org.elasticsearch.xpack.core.security.authc.AuthenticationField.AUTHENTICATION_KEY;
 
 /**
- * A service that exposes the CRUD operations for the async-search index.
+ * A service that exposes the CRUD operations for the async-task index.
```

Comment:
The name of the index is now passed as the first parameter of the AsyncTaskIndexService constructor, so it can be anything. AsyncSearch passes the original name of the index as the first parameter when it instantiates this class.

Reply:
I've seen that, maybe update the comment to clarify that async-task is not the name of the index?